beancount/scripts/directories_test.py

```diff
@@ -55,7 +55,8 @@ def test_validation(self):
                 any(
                     expected_account in message
                     for expected_account in expected_error_accounts
-                )
+                ),
+                f"{expected_error_accounts=!r} not in {message=!r}",
```

Comment:
This test error message is misleading as it can be read as the literal string to the left hand side of the `not in` operator is searched in the literal string on the right hand side. If you would like a better error message, the test can be rewritten as something like:
```python
for error in errors:
    for expected_account in expected_accounts:
        self.assertNotIn(expected_account, str(error))
```
or something like this.

Reply:
I reverted this line